src/sentry/tasks/relay.py

```diff
@@ -32,8 +32,7 @@ def build_project_config(public_key=None, **kwargs):
 
     Do not invoke this task directly, instead use :func:`schedule_build_project_config`.
     """
-    with configure_scope() as scope:
-        scope.set_tag("project.public_key", public_key)
+    sentry_sdk.set_tag("project.public_key", public_key)
```

Comment:
This was actually what I had in mind:
```suggestion
    sentry_sdk.set_tag("public_key", public_key)
```
`"project.public_key"` is not a bad name, but `"public_key"` is the name already used on line 233.